application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +46,12 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            if (event.getAuthor().isBot()) return;
+            message.delete().flatMap(any -> warnUser(message)).queue((res)->{
+                res.delete().queueAfter(1, TimeUnit.MINUTES);
+            },(err)->{
+
+            });
```

Comment:
and the most important part is to fix the actual bug. namely to make the code aware that sending a DM could fail and to instead send a log message instead of yielding an exception

Reply:
shall I make a 
```
try{
    //dm
}
catch{
    //ping
}

Reply:
no that won't work. Read the issue the solution is mentioned there

Reply:
I just need to DM the user if the DM fails I must make a ping and Inform the same message through the channel (Media channel) am I right now?

Reply:
again, u are misunderstanding the issue. the DM thing is a secondary feature. please first focus on the actual problem. the problem is that the code sends an exception when it fails to DM the user, instead of only sending a debug-log level message.

Reply:
(please dont resolve comments urself, wait for people to approve that u fixed it, thanks)

Reply:
undo all ur changes and then focus on the actual problem with the exception. use `onFailure` and let it send a debug log instead. then verify that u dont get an exception anymore.

Reply:
I did use it in the submission 5  just watch it out then